src/Schema/Comparator.php

```diff
@@ -190,6 +196,18 @@ public function diffSequence(Sequence $sequence1, Sequence $sequence2)
         return $sequence1->getInitialValue() !== $sequence2->getInitialValue();
     }
 
+    /**
+     * This method is deprecated and will be removed in the next major release.
```

Comment:
This usually describes what the method does, the users may see this in their IDEs, etc. The `@deprecated` annotation seems sufficient to me.